src/cogs/admin_commands.py

```diff
@@ -88,3 +91,32 @@ async def delete_finished_surveys_channels(self, ctx):
             deleted_channels.append(channel_id)
 
         await ctx.channel.send(f"Deleted {len(deleted_channels)} channels.")
+
+    @commands.command(name="create-channel-teams")
+    @commands.has_role(ADMIN_ROLE_ID)
+    async def create_channel_teams(self, ctx, channel_id, number_of_teams=2):
+        logging.info(
+            f"Executing create-channel-teams command. Channel ID: {channel_id}. Number of teams: {number_of_teams}."
+        )
+
+        channel = self.bot.get_channel(int(channel_id))
```

Comment:
Maybe add try catch or something like that? What if channel didn't exist?

Reply:
Good point. I fixed it. Check it out and feel free to merge it if it's ok. I accidentaly fixed the previous commit, instead of creating a fixup.